aiida/engine/processes/calcjobs/calcjob.py

```diff
@@ -744,15 +744,16 @@ def presubmit(self, folder: Folder) -> CalcInfo:
         priority = self.node.get_option('priority')
         if priority is not None:
             job_tmpl.priority = priority
+
+        max_memory_mb = computer.get_default_mem_per_machine()  # Default value, low priority
```

Comment:
Logic could be `if set on job, else if default, else None`

Reply:
done